core/src/main/java/com/taobao/arthas/core/command/monitor200/TimeTunnelCommand.java

```diff
@@ -167,6 +165,20 @@ public void setNumberOfLimit(int numberOfLimit) {
         this.numberOfLimit = numberOfLimit;
     }
 
+
+    @Option(shortName = "r", longName = "replayTimes")
```

Comment:
In my option, `replay times` and `replay interval` will only be used by advanced users, so `shortName`  is not needed.

Please use the `replay-times/replay-interval` style.